bin/node-template/node/src/command.rs

```diff
@@ -68,8 +69,7 @@ pub fn run() -> sc_cli::Result<()> {
 		Some(subcommand) => {
 			let runner = cli.create_runner(subcommand)?;
 			runner.run_subcommand(subcommand, |config| {
-				let (builder, _, _) = new_full_start!(config);
-				Ok(builder.to_chain_ops_parts())
+				crate::service::Builder::build_ops(config)
```

Comment:
I realize now that subcommand and ops are actually tight together and should probably share a name. Maybe we should rename `run_subcommand` to `run_ops`.

More context/info and thought: "ops" or something else, it doesn't matter much but it must clearly identifies the builtin operations we can do on the chain. The word "subcommand" is more generic and cli-oriented. `run_subcommand` is not actually used to run "any" subcommand, it's only for ops subcommands. Extra subcommands use `run_sync` and `run_async`